dts/bindings/timer/st,stm32-lptim.yaml

```diff
@@ -0,0 +1,5 @@
+description: STM32 lptim
```

Comment:
Please put the usual header:
```
# Copyright (c) 2020, STMicroelectronics
# SPDX-License-Identifier: Apache-2.0
```

Reply:
added also in the dts/bindings/timer/st,stm32-timers.yaml